src/loaders/features/enabled-features.js

```diff
@@ -1,20 +1,16 @@
-import { getConfigurationValue, getRuntime } from '../config/config'
+import { asyncFeatures, syncFeatures } from './features'
+import { getConfigurationValue, getRuntime } from '../../common/config/config'
 
-const featureNames = [
-  "ajax",
-  "jserrors",
-  "metrics",
-  "page_action",
-  "page_view_event",
-  "page_view_timing",
-  "session_trace",
-  "spa"
-]
+const featureNames = [...asyncFeatures, ...syncFeatures]
```

Comment:
I don't get the value of separating them.

Reply:
https://issues.newrelic.com/browse/NEWRELIC-6479